paddle/fluid/operators/math/blas.h

```diff
@@ -184,6 +184,11 @@ class Blas {
   template <typename T>
   void VINV(int n, const T* a, T* y) const;
 
+#ifdef PADDLE_WITH_MKLML
```

Comment:
Do not add `#ifdef` here.

You can make this function general.

Can refer to `VMUL`

Reply:
Done